### ISSUE TITLE
power higrow board through usb without battery?

### ISSUE THREAD

**Issue by Untiring** [Xinyuan-LilyGO/LilyGo-HiGrow#55]
since i found no information on this what so ever, i'd just like to check here if its possible to run the board without a battery connected?

**Comment by DanielBustillos** [comment]
Yes it is possible. You can use it by connecting it to an usb cable

**Comment by Untiring** [comment]
Alright thanks, then I now know my board is dead
